packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -1,72 +1,77 @@
 import { Box } from 'grommet'
-import { array, arrayOf, bool, shape, string } from 'prop-types'
+import { observer } from 'mobx-react'
+import { bool, string } from 'prop-types'
 import styled from 'styled-components'
-import { withResizeDetector } from 'react-resize-detector'
+import { useResizeDetector } from 'react-resize-detector'
 
-import ApprovedIcon from './components/ApprovedIcon'
-import Avatar from './components/Avatar'
-import Background from './components/Background'
-import DropdownNav from './components/DropdownNav'
-import LocaleSwitcher from './components/LocaleSwitcher'
-import Nav from './components/Nav'
-import ProjectTitle from './components/ProjectTitle'
-import UnderReviewLabel from './components/UnderReviewLabel'
+import {
+  ApprovedIcon,
+  Avatar,
+  Background,
+  DropdownNav,
+  LocaleSwitcher,
+  Nav,
+  ProjectTitle,
+  UnderReviewLabel
+} from './components/'
+import { useProjectNavigation, useStores } from './hooks/'
 
 const StyledBox = styled(Box)`
   position: relative;
 `
 
 function ProjectHeader({
-  availableLocales = [],
-  className = '',
-  inBeta = false,
-  navLinks = [],
-  title,
-  width
+  adminMode,
+  className = ''
 }) {
+  const { width, height, ref } = useResizeDetector()
+  const navLinks = useProjectNavigation(adminMode)
+  const {
+    availableLocales,
+    inBeta,
+    title
+  } = useStores()
+
   const hasTranslations = availableLocales?.length > 1
-  const showDropdownWithRow = (hasTranslations && width < 1200) || (!hasTranslations && width < 1100)
-  const showDropdownWithColumn = (hasTranslations && width < 1000) || (!hasTranslations && width < 900)
```

Comment:
These could both be true, which was confusing. On small screens, the header had both `showDropdownWithRow` and `showDropdownWithColumn` set, so it wasn't clear whether it should be using row or column layout. I've replaced this with Grommet's flexbox `direction`.